build-aux/snap/snapcraft.yaml

```diff
@@ -11,7 +11,7 @@ description: |
 adopt-info: snapd-deb
 # build-base is needed here for snapcraft to build this snap as with "modern"
 # snapcraft
-build-base: core
+build-base: core20
```

Comment:
I think this should go with a comment indicating this is specific to riscv branch and that there was no riscv support in 16.* or 18.* so 20.04 is the lowest common denominator of all supported riscv systems where snapd reexecs. (was there a riscv port of Debian 10?)

Reply:
In general, we want to move snapd snap to core20 anyway.

Even if there was riscv support on core & core18, I would not want to bootstrap riscv64 snaps using those old releases ever, given how old toolchains they have with little riscv64 support.

Eventually, all of the changes in this branch should go into master. Unfortunately we cannot QA and do them all yet. Nothing here is per-se riscv64 specific.

We will not be using build-base: core and snapcraft 4.x forever! =)